fleet/models.py

```diff
@@ -5,7 +5,6 @@
 
 
 class Station(models.Model):
-    station_id = models.CharField(max_length=2)
```

Comment:
What we discussed is that you should only use `station_id` for display purposes (but not for references), and you should use `pk` for references (but not for display purposes). The way it is right now is, you have zero control over what becomes a station 1, station 2, etc, and you can make a station 10 without first making 1, 2, 3, ..., 9.